utils/forms/forms.js

```diff
@@ -0,0 +1,368 @@
+import { loadCSS } from '../../scripts/aem.js';
+
+function buildLabel(field) {
+  const label = document.createElement('label');
+  label.textContent = field.required === true ? `${field.label} *` : field.label;
+  return label;
+}
+
+// Helper functions to show and clear error messages
+function showError(input, errorMessages) {
+  // Check if the error container already exists next to the input
+  let errorContainer = input.nextElementSibling;
+  if (!errorContainer || !errorContainer.classList.contains('error-messages')) {
+    // Create a new error container if one doesn't already exist
+    errorContainer = document.createElement('div');
+    errorContainer.className = 'error-messages';
+    input.insertAdjacentElement('afterend', errorContainer);
+  } else {
+    // Clear existing error messages if the container already exists
+    errorContainer.innerHTML = '';
+  }
+
+  // Append each error message as a separate span element
+  errorMessages.forEach((msg) => {
+    const span = document.createElement('span');
+    span.className = 'forms-error';
```

Comment:
does `.error-messages` container ever have any other content? is class necessary?